NEWS

```diff
@@ -32,6 +32,10 @@ CHANGES WITH 253 in spe:
 	  virtualized ones, and is activated in the case that the system token
 	  is missing from either sd-boot and sd-stub booted systems.
 
+	* systemd-userdbd and systemd-homed now honour credentials passed
+	  using SO_PASSCRED, and systemd-userdbd acting as multiplexer sends
+	  the credentials of the original connecting client to all services
```

Comment:
hmm, it does? i don't see where the ucred data is propagated

Reply:
Well, looking at the code again, it seems it really doesn't. I was under the impression that the same Varlink struct is re-used when talking to the services, and the debug logging actually confirms the credentials are correctly passed (systemd-homed logs my correct UID when I query the multiplexer). That's interesting…